src/cli/cmds/clone/handlers.js

```diff
@@ -0,0 +1,25 @@
+import getEnvironment from '../../environment';
+import { logYargs } from '../../yargs';
+import Clone from '../../../sub-commands/clone';
+
+const handlers = {
+  handleRun
+};
+
+export default handlers;
+
+function handleRun(argv, yargs, rawArgs = process.argv.slice(3)) {
```

Comment:
The `rawArgs` parameter can be removed as it isn't needed here in the same way as it was for `generate`.